modules/create-wif-cluster-ocm.adoc

```diff
@@ -0,0 +1,226 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-wif-cluster-ocm_{context}"]
+= Creating a Workload Identity Federation cluster using {cluster-manager}
+
+.Procedure
+
+. Log in to {cluster-manager-url} and click *Create cluster* on the {product-title} tile.
```

Comment:
Got confirmation from UXD and HCC style and terminology group that this UI element is a "card".
```suggestion
. Log in to {cluster-manager-url} and click *Create cluster* on the {product-title} card.
```